### PR TITLE
bugFix: Fix the whitespace issue

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: yaml.go [hunk 1]
```diff
@@ -167,7 +168,7 @@ func applyYaml(org string, fileName string, format string) {
 func makeCommaSeparatedString(arr []string) string {
 	var result string
 	for _, v := range arr {
-		result += v + ","
+		result += strings.TrimRight(v, " ") + ","
```
Comment (amirashad) [inline yaml.go]: Maybe use TrimSpace to trim spaces from leading and trailing both?

Comment (amirashad) [inline yaml.go]:
```suggestion
		result += strings.TrimSpace(v) + ","
```

Comment (ghost) [reply yaml.go]: Done. Thanks!

**Review by amirashad — APPROVED** [review]

**Comment by amirashad** [comment]
Thank you very much for your contribution

**Comment by ghost** [comment]
> Thank you very much for your contribution

I'm happy to help! :)

**Pull request merged**